doc/source/examples/incompressible-flow/2d-taylor-couette-flow-nitsche/2d-taylor-couette-flow-nitsche.rst

```diff
@@ -0,0 +1,358 @@
+========================================================
+2D Taylor-Couette Flow Using Nitsche Immersed Boundaries
+========================================================
+
+This is the XXe Lethe example. It revisits the same taylor-couette flow problem in `Example 2 <https://lethe-cfd.github.io/lethe/examples/incompressible-flow/2d-taylor-couette-flow/2d-taylor-couette-flow.html>`, 
```

Comment:
Third Lethe example, for now!

Reply:
Yes I was waiting for this PR to be ready to merge before udating this number! I will keep the comment open